.ci/scripts/travis_has_changes.sh

```diff
@@ -1,11 +1,14 @@
 #!/usr/bin/env bash
 set -exuo pipefail
 
-# commit range to check for. For example master...<PR branch>
+# Changes on these files will trigger all builds.
+COMMON_DIRLIST="vendor dev-tools .travis.yml"
```

Comment:
should we add `.ci` as well?

Reply:
Yeah, it would make sense, opened PR for that https://github.com/elastic/beats/pull/17387, thanks!